components/avatar/index.css

```diff
@@ -11,121 +11,165 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-Avatar {
-  --spectrum-avatar-color-opacity: 1;
+	--spectrum-avatar-color-opacity: 1;
 
-  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-100);
-  --spectrum-avatar-block-size: var(--spectrum-avatar-size-100);
+	--spectrum-avatar-inline-size: var(--spectrum-avatar-size-100);
+	--spectrum-avatar-block-size: var(--spectrum-avatar-size-100);
 
-  --spectrum-avatar-border-radius: var(--spectrum-avatar-block-size);
+	--spectrum-avatar-border-radius: var(--spectrum-avatar-block-size);
+	--spectrum-avatar-border-thickness: var(--spectrum-avatar-border-width);
+	--spectrum-avatar-border-color: rgb(var(--spectrum-gray-25-rgb));
```

Comment:
I actually went back and tried this with the token from the design, but changed the custom property name a touch, and I think everything is working ok now.

`--spectrum-avatar-border-color-default: var(--spectrum-avatar-border-color);`

I wonder if we had some caching issues? Because, it was definitely coming up undefined right? Maybe tweaking the custom property and adding that `-default` helped?

Reply:
So strange! I see it working now too. Just updated the PR. ✨